### PR TITLE
fix: deleted a line duplicating the call to the recognizer 

### PR DESCRIPTION
**Pull request by b-loved-dreamer**
Pushing this change to fix a duplicate issue

### CODE REVIEW

File: samples/microphone/transcribe_streaming_infinite.py [hunk 1]
```diff
@@ -41,9 +41,9 @@
 SAMPLE_RATE = 16000
 CHUNK_SIZE = int(SAMPLE_RATE / 10)  # 100ms
 
-RED = "\033[0;31m"
-GREEN = "\033[0;32m"
-YELLOW = "\033[0;33m"
+RED = '\033[0;31m'
+GREEN = '\033[0;32m'
+YELLOW = '\033[0;33m'
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:46]: Single quote marks won't pass our lint.  Please run `nox -s blacken`.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 2]
```diff
@@ -123,14 +123,12 @@ def generator(self):
                     if self.bridging_offset > self.final_request_end_time:
                         self.bridging_offset = self.final_request_end_time
 
-                    chunks_from_ms = round(
-                        (self.final_request_end_time - self.bridging_offset)
-                        / chunk_time
-                    )
+                    chunks_from_ms = round((self.final_request_end_time -
+                                            self.bridging_offset) / chunk_time)
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:127]: Again, please run `nox -s blacken`.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 3]
```diff
@@ -123,14 +123,12 @@ def generator(self):
...
 
-                    self.bridging_offset = round(
-                        (len(self.last_audio_input) - chunks_from_ms) * chunk_time
-                    )
+                    self.bridging_offset = (round((
+                        len(self.last_audio_input) - chunks_from_ms)
+                                                  * chunk_time))
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:131]: And here, too.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 4]
```diff
@@ -195,45 +193,42 @@ def listen_print_loop(responses, stream):
...
-        result_nanos = 0
+        result_micros = 0
 
         if result.result_end_time.seconds:
             result_seconds = result.result_end_time.seconds
 
-        if result.result_end_time.nanos:
-            result_nanos = result.result_end_time.nanos
+        if result.result_end_time.microseconds:
+            result_micros = result.result_end_time.microseconds
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:202]: This cannot be correct:  the [docs for `google.protobuf.Duration`](https://developers.google.com/protocol-buffers/docs/reference/google.protobuf#google.protobuf.Duration) show only a `nanos` field.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 5]
```diff
@@ -159,7 +157,7 @@ def generator(self):
                 except queue.Empty:
                     break
 
-            yield b"".join(data)
+            yield b''.join(data)
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:160]: Again, `nox -s lint`.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 6]
```diff
@@ -195,45 +193,42 @@ def listen_print_loop(responses, stream):
...
-        stream.result_end_time = int((result_seconds * 1000) + (result_nanos / 1000000))
+        stream.result_end_time = int((result_seconds * 1000) + (result_micros / 1000))
 
-        corrected_time = (
-            stream.result_end_time
-            - stream.bridging_offset
-            + (STREAMING_LIMIT * stream.restart_counter)
-        )
+        corrected_time = (stream.result_end_time - stream.bridging_offset
+                          + (STREAMING_LIMIT * stream.restart_counter))
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:207]: Again, `nox -s lint`.

File: samples/microphone/transcribe_streaming_infinite.py [hunk 7]
```diff
@@ -195,45 +193,42 @@ def listen_print_loop(responses, stream):
...
         # Display interim results, but with a carriage return at the end of the
         # line, so subsequent lines will overwrite them.
 
         if result.is_final:
 
             sys.stdout.write(GREEN)
-            sys.stdout.write("\033[K")
-            sys.stdout.write(str(corrected_time) + ": " + transcript + "\n")
+            sys.stdout.write('\033[K')
+            sys.stdout.write(str(corrected_time) + ': ' + transcript + '\n')
```
Comment (tseaver) [inline samples/microphone/transcribe_streaming_infinite.py:215]: Again, `nox -s lint`.

**Review by tseaver — DISMISSED** [review]
Please reduce the contents of this PR to those relevant to the changes you want to make.  We require that submitted code pass our linter, including having no changes when `black` is run using our configuration.

**Review by telpirion — COMMENTED** [review]
LGTM, pending all reviewer changes.

File: samples/microphone/transcribe_streaming_infinite.py (hunk 7 above)
Comment (b-loved-dreamer) [reply samples/microphone/transcribe_streaming_infinite.py]: These issues have been fixed.

**Comment by b-loved-dreamer** [comment]
Hi @tseaver, I've resolved all those issues. All checks have now passed.

**Review by sirtorry — APPROVED** [review]

**Pull request merged**